service/listeningService.go

```
@@ -37,20 +39,22 @@ func (s *CommandService) ListeningService(response http.ResponseWriter, request
 		bytePacket, err := dataPacket.ToByte()
 		if err != nil {
 			msg = "Failed to update your nickname."
-			response.WriteHeader(http.StatusInternalServerError)
+			logrus.Errorf("Failed to marshal message: %v", err)
+			utils.Errors.NewInternalError(response, err.Error())
 			return
 		}
 		if err := queue.SendMessage(bytePacket); err != nil {
 			msg = "Failed to update your nickname."
```

golangci-lint (1.62.2)

ineffectual assignment to msg

(ineffassign)